src/app/components/PageLayoutWrapper/index.tsx

```diff
@@ -73,6 +70,37 @@ const PageLayoutWrapper = ({
         if (!innerBlocks) return reducer;
         return reducer + innerBlocks.split(' ').length;
       }, 0);
+
+    const topStories = pageData.secondaryColumn?.topStories;
+    const mostReadItems = pageData.mostRead?.items;
+    const servicesForVariantA = ['pidgin', 'hindi', 'urdu', 'uzbek', 'hausa'];
+    const servicesForVariantB = ['mundo', 'burmese', 'arabic'];
+
+    let variantValue;
+
+    if (servicesForVariantA.includes(service)) {
+      variantValue = 'A';
+    } else if (servicesForVariantB.includes(service)) {
+      variantValue = 'B';
+    } else {
+      variantValue = 'none';
+    }
+    const experimentVariant: 'A' | 'B' | 'none' = ['A', 'B'].includes(
+      variantValue,
+    )
+      ? (variantValue as 'A' | 'B')
+      : 'none';
```

Comment:
Pretty sure this can all be controlled in Optimizely.

We can have separate 'rules' for the different services groupings and only show certain variations for the services in those groupings.

Reply:
Unless this is for testing only?

Reply:
Yes this is for testing only until we set up Optimizely. We can add the code that makes it work with Optimizely in another PR. I will need to know what name we will have for each variation.